### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by eliezercazares** [artilleryio/artillery#2037]
Version info: 
```
<version> 2.0.0-27
```

Running this command: 
```
<command> artillery run -e functional --output report.json ---config config.xml scenarios/scenario.yml
```

This is more of a question...

Can I use variables in the URL object?

I.e.

`
scenarios:
    - name: "My scenario"
       flow:
      -post:
           url: "/abc/def/{variable}"
`

thanks a lot for your help!
I have been able to use it in the company I work for because you guys have so much documentation and community. Thanks!

**Comment by hassy** [comment]
Yes @eliezercazares, you can use variables in URLs (they use double curly brackets, e.g. `/abc/{{ variable }}`.
